index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Paul Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+
+    <link 
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+        rel="stylesheet"
+    />
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <a href="#" class="logo">Paul</a>
+            <i class="material-icons">menu</i>
+        </nav>
+    </header>
+
+    <main>
+      <section class="section">
+      
+        <h1 class="main-title">Hey There.
+          I'm Paul</h1>
+        <h4 class="sub-title">I am a Software Developer</p>
+        <p class="description">I can help you build a product, feature or website. 
+          Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.
+        </p>
+        <div class="social-icons">
+          <ul class="social-wrapper">
+            <li>
+              <a href="#">
+                <img src="./images/github.png" alt="" class="social">
+              </a>
+            </li>
+  
+            <li>
+              <a href="#">
+                <img src="./images/linkedln.png" alt="" class="social">
```

Comment:
[ ] your LinkedIn icon is not showing up, kindly fix that .(i think you have an error in  `src="./images/linkedln.png"` the I in `Linkedin` is not capital).👇👇🤓

![linkedin](https://user-images.githubusercontent.com/62156607/198583672-e177ce12-5a4e-4e06-94ed-edd02f6f1b08.PNG)

[OPTIONAL ] for the best UI you have to use the same type of icons on your website.